### ISSUE TITLE
wrong url following the readme quickstart

### ISSUE THREAD

**Issue by asdf8601** [WaylonWalker/markata#17]
I've started using `markata` and following the quick-start from the readme I've found an issue with index's urls.

## MRE

```bash
mkdir pages
cd pages
echo '# My First Post' > first-post.md
echo '# Hello World' > hello-world.md
pipx run markata build
```

When I've opened the output `markout/index.html` both links didn't work. Here's the html code:

```html
<li class='post'>
<a href="/hello-world/">
    Hello World
</a>
</li>

<li class='post'>
<a href="/first-post/">
    First Post
</a>
```

I guess that maybe the relative path ( `.` ) is missed; what it works in my case:

```diff
  <li class='post'>
- <a href="/hello-world/">
+ <a href="./hello-world/index.html">
      Hello World
  </a>
  </li>
  
  <li class='post'>
- <a href="/first-post/">
+ <a href="./first-post/index.html">
      First Post
  </a>
```

**Comment by WaylonWalker** [comment]
Are you running a webserver or, just opening the HTML directly?  I just tried it on my machine and this works for me.

I did notice that you are running markata from the pages directory.  I will probably change the quickstart to this so that you run from the root directory.

``` bash
mkdir pages
echo '# My First Post' > pages/first-post.md
echo '# Hello World' > pages/hello-world.md
pipx run markata build
python -m http.server 8000 --directory markout
```
 
![image](https://user-images.githubusercontent.com/22648375/153907105-ae788f35-10ec-4ec4-9b6a-64876c1e7985.png)
